apps/prairielearn/assets/scripts/instructorInstanceAdminLti13Client.ts

```diff
@@ -0,0 +1,16 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  const selectLti13Instance = document.querySelector<HTMLSelectElement>('#selectLti13Instance');
+  const prefix = selectLti13Instance?.dataset?.urlPrefix;
+
+  if (!prefix) return;
+
+  selectLti13Instance?.addEventListener('change', () => {
+    const selectedOption = selectLti13Instance?.querySelector<HTMLOptionElement>('option:selected');
```

Comment:
`option:selected` isn't a valid selector. I'd recommend using the [`selectedOptions`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions) property on `selectLti13Instance` instead.